components/blog/PressRelease.vue

```diff
@@ -0,0 +1,137 @@
+<i18n>
+en:
+  title: The Go Conference of a programming language Go, will be held in Fukuoka on July 13
+  date: July 2, 2019
+  press:
+    to: 報道関係各位
+    date: 2019年7月2日
+    from: Go Conference運営委員会
+  body1: |-
+    プログラミング言語Go（以下、Go）に関する国内最大級のイベント「Go Conference'19 Summer in Fukuoka」が、来る7月13日（土）に、スタートアップで賑わうIT都市、福岡にて開催されます。
+  body2: |-
+    「Go Conference」は、2013年から開催し今年が7年目になります。今回は13回目のイベントとなり、「Go Conference」初めての地方開催を「エンジニアフレンドリーシティ」をうち出す福岡で行います。
+    Goは、プログラミング言語として後発です。そのため、Goは他の言語にないユニークな特徴を持ち、分散システムを実現するコンテナ技術関連のソフトウェアに採用されるなど、多くの人を魅了しています。
+    福岡のGoエンジニアはもちろん、東京をはじめ全国からの登壇者により全29セッションを予定しており、来場者は、150〜200名を見込んでおります。
+  summary:
+    title: カンファレンス概要
+    body:
+      - 名称：Go Conference'19 Summer in Fukuoka
+      - 日時：2019年7月13日（土）
+      - 会場：Fukuoka Growth Next（イベントホール、Startup Cafe）
+      - 参加費：無料
+      - 定員：200名
+      - 主催：Go Conference'19 Summer in Fukuoka 運営委員会
+      - 協賛：GMOペパボ、サーキュレーション、チームラボ、ディー・エヌ・エー、Fusic、ヌーラボ、メルカリ、アイキューブドシステムズ、LINE（申込み順）
+      - 公式サイト：https://fukuoka.gocon.jp/
+      - 申し込みサイト：https://fukuokago.connpass.com/event/130797/
+ja:
+  title: プログラミング言語Goの祭典「Go Conference」が7月13日に福岡で開催
+  date: '2019-07-02'
+  body1: |-
+    プログラミング言語Go（以下、Go）に関する国内最大級のイベント「Go Conference'19 Summer in Fukuoka」が、来る7月13日（土）に、スタートアップで賑わうIT都市、福岡にて開催されます。
```

Comment:
下の行でも福岡への修飾があり，文の主題がぼやけるため，ここではカンファレンスが開催されること，場所，日付を述べるに留めて見てはどうでしょうか

```suggestion
プログラミング言語Go（以下、Go）に関する国内最大級のイベント「Go Conference'19 Summer in Fukuoka」が、2019年7月13日（土）に福岡で開催されます。
```

Reply:
良さ